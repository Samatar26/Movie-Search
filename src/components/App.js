import React, { Component } from 'react';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';
import reduxThunk from 'redux-thunk';
import SearchForm from './../containers/SearchForm';
import Movies from './../containers/Movies.js';
import reducers from '../reducers';
import '../styles/reset';

const createStoreWithMiddleware = applyMiddleware(reduxThunk)(createStore);



class App extends Component {
  render() {
    return (
     <Provider store={createStoreWithMiddleware(reducers)}>
      <div>
        <SearchForm />
        <Movies />
      </div>
    </Provider>
    );
  }
}

export default App;
