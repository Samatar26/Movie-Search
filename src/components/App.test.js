import React from 'react'
import App from './App'
import CoreTestWrapper from '../config/test-wrapper'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <CoreTestWrapper>
      <App/>
    </CoreTestWrapper>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})