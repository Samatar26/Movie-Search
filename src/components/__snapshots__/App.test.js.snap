// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    className="sc-bwzfXH eTiFGk"
    onSubmit={[Function]}
  >
    <input
      className="sc-bdVaJa kYdqnO"
      name="searchTerm"
      onBlur={[Function]}
      onChange={[Function]}
      onDragStart={[Function]}
      onDrop={[Function]}
      onFocus={[Function]}
      type="text"
      value=""
    />
  </form>
  <main
    className="sc-htpNat gcwEVk"
  />
</div>
`;
